doc/esmvalcore/recipe.rst

```diff
@@ -47,25 +56,34 @@ recipe_ocean_amoc.yml.
       projects:
         - ukesm
 
-Note that the authors, projects, and references will need to be included in the
-``config-references.yml`` file. The author name uses the format:
-`surname_name`. For instance, Mickey Mouse would be: `mouse_mickey`.
-Also note that this username is unlikely to be the same as the github
-user name.
+.. note::
 
+   **Information from config-references.yml**
 
+   Note that the authors, projects, and references will need to be included in the
+   ``config-references.yml`` file. The author name uses the format:
+   ``surname_name``. For instance, Joe Pesci would be: ``authors: - pesci_joe``.
```

Comment:
I would not use a real person name here.

Reply:
hahah, just an example, nobody's gonna use Joe Pesci :grin:

Reply:
John Doe would be better